src/plugins/lite/plugin.ts

```diff
@@ -128,7 +128,11 @@ const LITE = {
   IS_LITE_CLASS_RE = new RegExp(
     '(?:^|s)(?:' + LITEConstants.deleteClass + '|' + LITEConstants.insertClass + ')(?:s|$)'
   ),
-  defaultTooltipTemplate = '%a by %u %t',
+  defaultTooltipDisplay = (change: any) => {
+    const action = 'insert' === change.type ? 'Added' : 'Deleted';
```

Comment:
I think is better to have lang.CHANGE_TYPE_ADDED : lang.CHANGE_TYPE_DELETED or at least using a defined variables